frontend/src/pages/UserProfile.jsx

```diff
@@ -206,13 +207,13 @@ function UserProfile(props) {
                             )}
                         </ul>
 
-                        <div className="webUrl">
```

Comment:
Could you discard your change to UserProfile.jsx? It is deleting my previous update.